db/migrate/20200322182816_create_jams.rb

```diff
@@ -0,0 +1,11 @@
+class CreateJams < ActiveRecord::Migration[6.0]
+  def change
+    create_table :jams do |t|
+      t.string :filename
+      t.string :bpm
```

Comment:
any reason for it to be a string vs integer?

Reply:
The thought was that this could maybe be a bit of a loose field, "Verse is 100bpm then chorus is 120bpm"